weblogic/tests/compose/weblogic/sample/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,4 @@
+Manifest-Version: 1.0

```

Comment:
Are all those `compose/weblogic/*` files needed to make the environment start?

Reply:
Yes they're for the E2E. The size is about 100kb. Do you think I should remove it and put it in `docker-library` repo?

Reply:
I can see if I can remove some of it or have it be downloaded during env start.

Reply:
I removed most of the scripts and instead download them in the Dockerfile during image build: https://github.com/DataDog/integrations-core/pull/10559/files#diff-8b0e9dac937af40127c0930e9069361d6b5296a3fbf0c02010cd77da01877098R54-R61